### PR TITLE
AV-56370:heat docker image for rocky.

### PR DESCRIPTION
**Pull request by Rishabh-Baranwal**

### CODE REVIEW

**Comment by Rishabh-Baranwal** [comment]
@anantpatil please review

File: rocky-docker/install_scripts/keystone.conf [hunk 1]
```diff
@@ -0,0 +1,2952 @@
+[DEFAULT]
```
Comment (anantpatil) [inline rocky-docker/install_scripts/keystone.conf]: Please add only relevant content here like in other files.

Comment (Rishabh-Baranwal) [reply rocky-docker/install_scripts/keystone.conf]: done

**Comment by Rishabh-Baranwal** [comment]
@ypraveen please review and merge

**Comment by ypraveen** [comment]
@Rishabh-Baranwal I am assuming this passes with heat tests?

**Comment by Rishabh-Baranwal** [comment]
@ypraveen yes

**Pull request merged**